Test-NetStack.psm1

```diff
@@ -406,14 +457,11 @@ Function Test-NetStack {
                         $Result | Add-Member -MemberType NoteProperty -Name RxPctgOfLinkSpeed -Value $thisTargetResult.ReceivedPctgOfLinkSpeed
                         $Result | Add-Member -MemberType NoteProperty -Name MinExpectedPctgOfLinkSpeed -Value $Definitions.TCPPerf.TPUT
 
-                        #$ThroughputPercentageDec = $Definitions.TCPPerf.TPUT / 100.0
-                        #$AcceptableThroughput = $thisSourceResult.RawData.MinLinkSpeedbps * $ThroughputPercentageDec
-
                         if ($thisTargetResult.ReceivedPctgOfLinkSpeed -and $Definitions.TCPPerf.TPUT) {
                             if ($thisTargetResult.ReceivedPctgOfLinkSpeed -ge $Definitions.TCPPerf.TPUT) { $Result | Add-Member -MemberType NoteProperty -Name PathStatus -Value 'Pass' }
                             else { $Result | Add-Member -MemberType NoteProperty -Name PathStatus -Value 'Fail' }
                         }
-                        else { throw 'Internal Error: Test-NetStack Data Integrity Issue in Stage2' }
+                        else { $Result | Add-Member -MemberType NoteProperty -Name PathStatus -Value 'Fail' }
```

Comment:
Same as above; the 'Fail' is correct, but the we should add the reason the the log file so we can distinguish this code path from a regular failure